app/uk/gov/hmrc/fileupload/controllers/FileController.scala

```diff
@@ -25,18 +25,33 @@ import uk.gov.hmrc.fileupload.read.envelope.{Envelope, WithValidEnvelope}
 import uk.gov.hmrc.fileupload.read.file.Service._
 import uk.gov.hmrc.fileupload.write.envelope.{EnvelopeCommand, EnvelopeNotFoundError, FileNotFoundError, StoreFile}
 import uk.gov.hmrc.fileupload.write.infrastructure.{CommandAccepted, CommandNotAccepted}
-import uk.gov.hmrc.play.microservice.controller.BaseController
+import uk.gov.hmrc.fileupload._
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
 
+
+package object file {
+
+  import play.api.libs.concurrent.Execution.Implicits._
```

Comment:
You may well be onto something there, however this hasn't really been introduced as part of the migration to 2.5. The controllers were previously initialised in MicroServiceGlobal which was importing the same ExecutionContext. If anything it is now easy to use a different context as the controllers are created in different scopes, each with their own import.